backend/api/helper/permissions.py

```diff
@@ -0,0 +1,16 @@
+from flask_security import current_user
+
+
+def current_user_has_permission_to_change(entry) -> bool:
```

Comment:
It was more versatily but permissions are allways checked against the current user